### PR TITLE
IGNORE

### PR DESCRIPTION
**Pull request by shaiy310**
TEST - IGNORE

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# Shahaf look at me
```
Comment (amshamah419) [inline __init__.py:1]: Please ensure coverage with UTs.

Comment (amshamah419) [inline __init__.py:1]:
```suggestion
# Shahaf, please look at me.
```

Comment (amshamah419) [reply __init__.py:1]: This also implies the existence of a `Shahaf` at all times. Consider making a variable.

Comment (yaakovi) [reply __init__.py:1]: > This also implies the existence of a `Shahaf` at all times. Consider making a variable.

he is here to stay

Comment (jochman) [inline __init__.py:1]: Look at my horse, my horse is amazing 🐎

Comment (ShahafBenYakir) [reply __init__.py:1]: 🤔

**Pull request closed without merging**